### PR TITLE
DashR doc updates to in-progress sections

### PR DESCRIPTION
**Pull request by KPhans**
Post-merge checklist:

The master branch is auto-deployed to `dash.plot.ly`.
Once you have merged your PR, wait 5-10 minutes and check dash.plot.ly 
to verify that your changes have been made.

- [ ] I understand

### CODE REVIEW

**Comment by KPhans** [comment]
fixed examples that were previously malfunctioning as well. fixed js and search index for algolia implementation. @cldougl will look into it as soon as she can.

**Comment by rpkyle** [comment]
@KPhans Thanks for this PR, having a look now!

File: run.R [hunk 1]
```diff
@@ -177,7 +167,7 @@ app$callback(
       '/dash-core-components/Input' = return(chapters.Input$layout),
       '/dash-core-components/Textarea' = return(chapters.TextArea$layout),
       '/dash-core-components/Checklist' = return(chapters.Checklist$layout),
-      '/dash-core-components/RadioItems' = return(chapters.RadioItems$layout),
+      '/dash-core-components/Radioitems' = return(chapters.RadioItems$layout),
```
Comment (rpkyle) [inline run.R]: This should remain `RadioItems`, as this was part of the issue previously with the path naming. I'll make this change and push.

File: run.R [hunk 2]
```diff
@@ -203,18 +193,22 @@ app$callback(
...
-
+            
+            
             components$Section(
               'What\'s Dash?',
               list(
                 components$Chapter(
                   'Introduction',
-                  href='https://dash.plot.ly/introduction',
+                  href='/introduction',
```
Comment (rpkyle) [inline run.R]: Similarly, want to keep this `/introduction` rather than `/Whats_dash` :slightly_smiling_face:

**Pull request merged**